lang/shared/src/main/scala/com/wavesplatform/lang/ParserImpl.scala

```diff
@@ -31,6 +31,13 @@ abstract class ParserImpl { this: Base58 =>
   private def someP: P[SOME]           = P("Some" ~ "(" ~ block ~ ")").map(x => SOME(x))
   private def noneP: P[NONE.type]      = P("None").map(_ => NONE)
 
+  private def functionCallArgs : P[Seq[EXPR]] = (expr.rep(exactly=1) ~ P("," ~ expr).rep()).map {
```

Comment:
1. Will we have functions without arguments?
2. Could you try `expr.rep(min = 1, sep = ",")` instead of `(expr.rep(exactly=1) ~ P("," ~ expr).rep())`?

Reply:
1. don't know yet
2. yes, good idea. didn't know about `sep`

Reply:
end up with `expr.rep(min = 0, sep = ",")`, much better